src/main/java/luna/week2/Question1.java

```diff
@@ -0,0 +1,11 @@
+package luna.week2;
+
+public class Question1 {
+
+    public static void main(String[] args) {
+        System.out.println("test case 1 added here");
```

Comment:
method name can be changed to compare :)

Reply:
What is redundant?